tidb-cloud/tidb-cloud-sso-authentication.md

```diff
@@ -0,0 +1,47 @@
+---
+title: SSO Authentication
+summary: Learn how to log into the TiDB Cloud console via your Google account or GitHub account.
+---
+# SSO Authentication
+
+This document describes how to log in to the [TiDB Cloud console](https://tidbcloud.com/) via Single Sign-on (SSO) authentication, which is quick and convenient.
+
+TiDB Cloud supports SSO authentication for Google and GitHub accounts. If you log into TiDB Cloud via SSO authentication, because your ID and credentials are stored on the third-party Google and GitHub platforms, you will not be able to modify your account password and enable multi-factor authentication (MFA) using the TiDB console. 
```

Comment:
```suggestion
TiDB Cloud supports SSO authentication for Google and GitHub accounts. If you log in to TiDB Cloud via SSO authentication, because your ID and credentials are stored on the third-party Google and GitHub platforms, you will not be able to modify your account password and enable multi-factor authentication (MFA) using the TiDB console. 
```